### PR TITLE
create function + partial doc

### PR DESCRIPTION
**Pull request by artdrou**

### CODE REVIEW

File: lib/audiofile.py [hunk 1]
```diff
@@ -49,3 +49,26 @@ def read(filePath, makeMono=False):
...
+def convertMonoDatatoBytes(x, rate, filename):
+    """ Convert synthetic signal x to bytes object readable by readframes function of wave module
+    Args:
+        [list]:     x,          signal data
+        [int]:      rate,       sampling frequency of the signal
+        [string]:   filename,   name of the wavfile that will be created
+    Returns:
+        [???]:      bytesData
+    """
+    import wave, struct
```
Comment (akpascalp) [inline lib/audiofile.py:63]: import at top of file

File: lib/audiofile.py [hunk 2]
```diff
@@ -49,3 +49,26 @@ def read(filePath, makeMono=False):
 def openWithFfmepg(filePath):
     # TODO finish this shit
     cmd = f'ffmpeg -i {filePath}'
+
+
+def convertMonoDatatoBytes(x, rate, filename):
+    """ Convert synthetic signal x to bytes object readable by readframes function of wave module
+    Args:
+        [list]:     x,          signal data
```
Comment (akpascalp) [inline lib/audiofile.py:57]: I prefer explicit variable name (and if possible more than 1 letter), except if for acoustic engineer it's a standard

File: lib/audiofile.py [hunk 3]
```diff
@@ -49,3 +49,26 @@ def read(filePath, makeMono=False):
...
+        [string]:   filename,   name of the wavfile that will be created
+    Returns:
+        [???]:      bytesData
+    """
+    import wave, struct
+    obj = wave.open(filename,'w')
+    obj.setnchannels(1) # mono
+    obj.setsampwidth(2)
+    obj.setframerate(rate)
+    for i, value in enumerate(x):
```
Comment (akpascalp) [inline lib/audiofile.py:68]: `for value in x:`

File: lib/audiofile.py [hunk 4]
```diff
@@ -49,3 +49,26 @@ def read(filePath, makeMono=False):
...
+        data = struct.pack('<h', int(value*32767))
+        obj.writeframesraw( data )
```
Comment (akpascalp) [inline lib/audiofile.py:70]: `obj.writeframesraw(data)`

**Comment by akaRicane** [comment]
agree with all @akpascalp comments (y)

**Comment by akaRicane** [comment]
@artdrou can you add opposite function ? e.g. from wave object to array of data ?

**Comment by akaRicane** [comment]
@akpascalp PR won't be merged yet, cause too much potential ! We will work together with @artdrou in order to convert all real time process in python objects in order to populate much wisely lib. 
PR should be ready by nex week

**Review by akaRicane — CHANGES_REQUESTED** [review]
cf comments

**Pull request closed without merging**